sdks/python/apache_beam/ml/transforms/base.py

```diff
@@ -134,7 +134,7 @@ def __init__(
     themselves.
 
     Args:
-      artifact_location: A storage location for artifacts resulting from
+      write_artifact_location: A storage location for artifacts resulting from
```

Comment:
Maybe add a note that this will overwrite any other artifacts in this location, and distinct artifact locations should be used for distinct ML transforms?

Reply:
Maybe mention that this should be a directory, and that it will be created if it does not already exist?